src/priority_queue.c

```diff
@@ -51,9 +54,20 @@ static size_t raise(priorq_t *prq, size_t slot, const void *value)
 
 void priorq_enqueue(priorq_t *prq, const void *value)
 {
+    if (prq->capacity == prq->max_capacity) {
+        size_t n = 1;
```

Comment:
why start from 1 instead of from `MAX(1, prq->max_capacity)`?

Reply:
Because it is nicer to read and saving a few iterations does not provide any significant benefit.